synapse/config/password_auth_providers.py

```diff
@@ -15,13 +15,15 @@
 
 from ._base import Config, ConfigError
 
-import importlib
+from synapse.util.module_loader import load_module
 
 
 class PasswordAuthProviderConfig(Config):
     def read_config(self, config):
         self.password_providers = []
 
+        provider_config = None
```

Comment:
Why are we doing this?

Reply:
just to make it clear the variable is at this level  (even though they all get hoisted)

Reply:
we don't usually bother, but ok